scripts/verify-jenkins-on-openshift.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+# This script deploys the jenkins pod with provided image (from CPaas) on an 
+# OpenShift cluster & executes verify_jenkins.sh within the newly deployed pod.
+set -o pipefail
+
+# Takes input & stores in the respective variables
+# 
+get_input(){
+    read -p "OpenShift's API URL: " API_SERVER
+    read -p "OpenShift's User Name (kubeadmin): " USER_NAME
+    read -p "OpenShift's User Password: " USER_PASSWORD
+    read -p "Jenkins Image built by CPaas: " JENKINS_IMAGE  
+}
```

Comment:
This seems like a pain to input one at a time, how about

1. Eliminate the API_SERVER, USER_NAME, USER_PASSWORD
2. Have the user do an export KUBECONFIG
3. have the JENKINS_IMAGE and SHA either be positional arguments to the script, or ENV VARS that are specified (similar to how the verify script does it)

Reply:
suree, eliminating those several inputs, would be a better option. On it.